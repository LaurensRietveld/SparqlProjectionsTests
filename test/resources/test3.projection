[
  {
    "iri": ?entity,
    "label": ?label,
    "extraInfo": [
      ?aPred:?anObj
    ]
  }
]
