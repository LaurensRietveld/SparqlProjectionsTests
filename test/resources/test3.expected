[
  {
    "iri": "http://dbpedia.org/resource/Aa_(architect)",
    "label": "Aa (Architekt) ",
    "extraInfo": [
      "http://www.w3.org/1999/02/22-rdf-syntax-ns#type": 	"http://www.w3.org/2002/07/owl#Thing",
      "http://www.w3.org/1999/02/22-rdf-syntax-ns#type": 	"http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#NaturalPerson",
      "http://www.w3.org/1999/02/22-rdf-syntax-ns#type": 	"http://www.wikidata.org/entity/Q215627",
      "http://www.w3.org/1999/02/22-rdf-syntax-ns#type": 	"http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Agent"
    ]
  }
]
  {
    "iri": "http://dbpedia.org/resource/Abel,_King_of_Denmark",
    "label": 	"Abel, King of Denmark",
    "extraInfo": [
    "http://www.w3.org/1999/02/22-rdf-syntax-ns#type": 	"http://www.w3.org/2002/07/owl#Thing",
    "http://www.w3.org/1999/02/22-rdf-syntax-ns#type": 	"http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Agent",
    "http://www.w3.org/1999/02/22-rdf-syntax-ns#type": 	"http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#NaturalPerson",
    "http://www.w3.org/1999/02/22-rdf-syntax-ns#type": 	"http://www.wikidata.org/entity/Q215627"

    ]

  }
]
